libs/langchain/langchain/tools/sql_database/prompt.py

```diff
@@ -11,4 +11,8 @@
 - Casting to the correct data type
 - Using the proper columns for joins
 
-If there are any of the above mistakes, rewrite the query. If there are no mistakes, just reproduce the original query."""
+If there are any of the above mistakes, rewrite the query. If there are no mistakes, just reproduce the original query.
+
+Output the final SQL query only.
+
+SQL Query: """
```

Comment:
@keenborder786 updated the prompt slightly, what do you think?

Reply:
@baskaryan please merge.

Reply:
@baskaryan please merge this, this is really important. Thank you.